ios/Video/RCTVideo.swift

```diff
@@ -1717,21 +1764,24 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
         }
     }
 
+    @available(tvOS 14.0, *)
```

Comment:
I think the goal of this change is to fix build issue on last release.
It has been fixed in another way, can you please revert it ?